### ISSUE TITLE
train loss inf

### ISSUE THREAD

**Issue by DamperHa** [yjn870/RCAN-pytorch#1]
Hello, did you encounter training loss as inf at the beginning of training RCAN?thx

**Comment by Edwardyan1112** [comment]
I had encounter this situation. How to fix this problem? thx
